ebml_matroska.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<EBMLSchema docType="matroska" version="4">
+ <?xml version="1.0" encoding="utf-8"?>
+<EBMLSchema docType="matroska" version="4" xmlns="https://ietf.org/cellar/ebml">
```

Comment:
Not sure it works for you but adding this makes the XML to MD/HTML not produce the right output (looking for a fix).
It is necessary for XSD validation so we should have it.

Reply:
I'll fix that in the xsl

Reply:
fixed and force pushed

Reply:
arf, I saw this after I did #328. IMO it's cleaner to use `ebml:` rather than `e:`